compiler/backend_inkwell/src/lib.rs

```diff
@@ -158,53 +156,53 @@ impl<'ctx> CodeGen<'ctx> {
         };
 
         self.builder.position_at_end(block);
-        self.compile_mir(&self.mir.body.clone(), &main_info);
+        let main_return = self
+            .compile_mir(&self.mir.body.clone(), &main_info)
+            .expect("Main Function should return a struct");
```

Comment:
Modules always return a struct of their exported things.

```suggestion
            .unwrap();
```